Resting breathing rates for college-age students are approximately normally distributed with
mean 12 and standard deviation 2.3 breaths per minute. What fraction of all college-age students
have breathing rates in the following intervals?

In [1]:
# A few values that need to be calculated first before answering the next series of questions

mean = 12
std = 2.3

interval_1 = [mean - std, mean + std]
print interval_1 

interval_2 = [mean - 2 * std, mean + 2 * std]
print interval_2

interval_3 = [mean - 3 * std, mean + 3 * std]
print interval_3

[9.7, 14.3]
[7.4, 16.6]
[5.1000000000000005, 18.9]


A) 9.7 to 14.3 breaths per minute

.68

B) 7.4 to 16.6 breaths per minute

.95

C) 9.7 to 16.6 breaths per minute

In [2]:
interval_difference_divided_in_half = (.95 - .68) / 2
internavl_containing_extra_interval = .68 + interval_difference_divided_in_half 
print internavl_containing_extra_interval

0.815


D) Less than 5.1 or more than 18.9 breaths per minute

.01